misc/scripts/models-as-data/bulk_generate_mad.py

```diff
@@ -15,7 +15,12 @@
 import requests
 import zipfile
 import tarfile
-from functools import cmp_to_key
+
+try:
+    import yaml
+except ImportError:
+    print("ERROR: PyYAML is not installed. Please install it with 'pip install pyyaml'.")
+    sys.exit(1)
```

Comment:
I hit a similar problem with `requests` when I first ran this script, FWIW.

Reply:
good point, I'll add that as well